apps/signing/serializers.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+from django.core.urlresolvers import reverse
+
+from rest_framework import serializers
+
+import amo
+from amo.helpers import absolutify
+from files.models import FileUpload
+
+
+class FileUploadSerializer(serializers.ModelSerializer):
+    active = serializers.SerializerMethodField('get_active')
+    api_url = serializers.SerializerMethodField('get_api_url')
+    files = serializers.SerializerMethodField('get_files')
+    passed_review = serializers.SerializerMethodField('get_passed_review')
+    processed = serializers.CharField(source='processed')
+    reviewed = serializers.SerializerMethodField('get_reviewed')
+    validation_results = serializers.SerializerMethodField(
+        'get_validation_results')
+    validation_url = serializers.SerializerMethodField('get_validation_url')
+
+    class Meta:
+        model = FileUpload
+        fields = [
+            'active',
+            'api_url',
+            'files',
+            'passed_review',
+            'processed',
+            'reviewed',
+            'valid',
+            'validation_results',
+            'validation_url',
+            'version',
+        ]
+
+    def __init__(self, *args, **kwargs):
+        self.version = kwargs.pop('version', None)
+        super(FileUploadSerializer, self).__init__(*args, **kwargs)
+
+    def get_api_url(self, instance):
```

Comment:
I originally put it in thinking that way you know what URL to poll but it's the same as the URL you just hit so that isn't super useful.

We should have URLs for things we return but we should be consistent and this might not be the best name. It looks like [HyperlinkedModelSerializer](http://tomchristie.github.io/rest-framework-2-docs/api-guide/serializers#hyperlinkedmodelserializer) uses `url` so I'll get it to use that.